pkg/cmd/cmd.go

```diff
@@ -61,11 +67,32 @@ func NewPatchCmd() *cobra.Command {
 				Platforms:     ua.platform,
 				Progress:      progressui.DisplayMode(ua.progress),
 				Loader:        ua.loader,
+				ConfigFile:    ua.configFile,
 			}
-			return Patch(context.Background(), opts)
+
+			if ua.configFile == "" && ua.appImage == "" {
+				return errors.New("either --config or --image must be provided")
+			}
+
+			// bulk patch
+			if ua.configFile != "" {
+				if ua.appImage != "" || ua.report != "" || ua.patchedTag != "" {
+					return errors.New("--config cannot be used with --image, --report, or --tag")
+				}
+
+				log.Info("Starting in bulk image patching mode...")
+
+				return bulk.PatchFromConfig(context.Background(), ua.configFile, opts)
+			}
+			if ua.appImage == "" {
+				return errors.New("--image is required when not using --config")
+			}
+			log.Info("Starting in single image patching mode...")
+			return patch.Patch(context.Background(), opts)
 		},
 	}
 	flags := patchCmd.Flags()
+	flags.StringVar(&ua.configFile, "config", "", "Path to a bulk patch YAML config file (Comprehensive update only). If used, --image and --report are ignored.")
```

Comment:
nit: it looks like it is not ignored but instead an error is thrown, in that case should we reword this to something like "Cannot be used with --image, --report, or --tag."?